scripts/config/jest/index.js

```diff
@@ -0,0 +1,21 @@
+const path = require('path');
```

Comment:
I see it another way, all of our code should be in `src`, but we put "pointers" into scripts so they are easier to access from the CLI.